decentraland/unity-explorer
# fix: added bus for mac audio event for media player

## What does this PR change?

Fix #2294 
This PR adds a workaround to control directly the media stream audio instead of relying on the AudioMixer values.
This is needed on macos because mac os doesn’t allow to route audio from HLS through to Unity. This is a limitation of Apple’s AVFoundation framework.

...

## How to test the changes?



1. Launch the explorer
2. Go to dollhouse
3. Verify that changing the world volume in the settings affects the volume of the scene stream

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
